### PR TITLE
hello-rails-react-app

### PR DESCRIPTION
**Pull request by Bifabig**
### In this PR i have done the following:

- Create a new Rails app called 'hello-rails-react'.
- Verify that the jsbundling-rails gem is added to Gemfile.
- Initialize project with Git.
- Install needed node dependencies.
- Create a babel.config.js in the root folder of the project.
- Update webpack.config.js to include the babel loader.
- Make sure that the project has a Postgres database set up.
- Create a table for storing messages and create 5 different greetings.
- Create an API endpoint that selects a random greeting from messages table.
- Create a static view that will be the root of the app.
- Create the App component with react-router and render it in a static view.
- Create the Greeting component that will display a greeting.
- Create a store, an action, and a reducer that will connect to API endpoint to get the random greeting.
- Display the random greeting in Greeting component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,175 @@
...
+  bundle install
+  npm i
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+   ./bin/dev
```
Comment (BregornOriginal) [inline README.md]: - [x] Kindly make your command `rails ./bin/dev` works, while running this one nothing happens

image

File: README.md [hunk 2]
```diff
@@ -1,24 +1,175 @@
...
+```
+
```
Comment (BregornOriginal) [inline README.md:109]: - [x] Kindly fix the error while running `rails s`, running your app with this command gives the following error

image

**Review by BregornOriginal — CHANGES_REQUESTED** [review]
Hi @Bifabig  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

giphy (4)

## Highlights

✅ Linters are passing
✅ Good commit history

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Bifabig** [comment]
> Hi @Bifabig
> 
> Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
> 
> giphy (4) [ giphy (4) ](https://user-images.githubusercontent.com/93630700/268963158-7625c904-c2af-48f3-a999-693ef32415a5.gif) [ ](https://user-images.githubusercontent.com/93630700/268963158-7625c904-c2af-48f3-a999-693ef32415a5.gif)
> 
> ## Highlights
> ✅ Linters are passing ✅ Good commit history
> 
> ## Required Changes ♻️
> _Check the comments under the review._
> 
> ## Optional suggestions
> _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
> 
> Cheers and Happy coding!👏👏👏
> 
> Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thanks @BregornOriginal it's now fixed. You can check the readme file.

**Review by IngDavidTM — APPROVED** [review]
Hi @Bifabig 🙋‍♂️, 

# ✔️   Status: APPROVED 🏆 🎉 🎊 

You have done a great job on this project, keep it up 👍 🥇. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations!  🎉💯🌟 

## Top highlight 🎯

- The Pull Request follows the GitFlow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**